src/main/java/cz/smarteon/loxone/LoxoneWebSocketListener.java

```diff
@@ -9,4 +9,11 @@
      * Loxone web socket has been opened.
      */
     void webSocketOpened();
+
+    /**
+     * Loxone web socket has been closed.
+     */
+    default void webSocketRemoteClosed(int code){
```

Comment:
Please document the parameter.

Please let the default implementation just empty. Calling println in thread handling the websocket connection could cause unexpected behaviour.

Would be nice to add simple test to `LoxoneWebSocketTest` - it will look similar to the `should call websocket listener` test.

Thanks

Reply:
I have already made the requested changes.